src/main/java/com/sparkplatform/api/models/Listing.java

```diff
@@ -357,10 +356,10 @@ public String getSubdivisionName() {
 		public void setSubdivisionName( String subdivisionName ) {
 			this.subdivisionName = subdivisionName;
 		}
-		public String getmLSAreaMinor() {
+		public String getMLSAreaMinor() {
 			return mLSAreaMinor;
 		}
-		public void setmLSAreaMinor( String mLSAreaMinor ) {
+		public void setMLSAreaMinor( String mLSAreaMinor ) {
```

Comment:
This was resulting in a test failing that looked for `isMLSAreaMinor`.  While I get why the attribute is named `mLSAreaMinor` instead of `MLSAreaMinor`, other fields like `subdivisionName` capitalize the first letter of the attribute, as I'd expect.

This may have been a copy-and-paste mistake in the first place.